maat-court-data-api/src/main/java/gov/uk/courtdata/hardship/controller/HardshipReviewController.java

```diff
@@ -64,6 +64,21 @@ public ResponseEntity<HardshipReviewDTO> getHardshipByRepId(@PathVariable int re
         return ResponseEntity.ok(hardshipReviewService.findHardshipReviewByRepId(repId));
     }
 
+    @GetMapping(value = "repId/{repId}/detailType/{detailType}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Retrieve a hardship review record by repId and detail type")
+    @ApiResponse(responseCode = "200", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = HardshipReviewDTO.class)))
```

Comment:
Many of these duplicated boilerplate API annotations can be replaced with a custom annotation see `@StandardApiResponseCodes` and the Controller `EformStagingController` for an example of usage.

https://github.com/ministryofjustice/laa-maat-court-data-api/blob/d8286767c4347df1e81fe6c99bcf5ad07de22379/maat-court-data-api/src/main/java/gov/uk/courtdata/eform/controller/StandardApiResponseCodes.java

https://github.com/ministryofjustice/laa-maat-court-data-api/blob/d8286767c4347df1e81fe6c99bcf5ad07de22379/maat-court-data-api/src/main/java/gov/uk/courtdata/eform/controller/StandardApiResponseCodes.java